### ISSUE TITLE
Virtual Hardware Key iOS Autofill

### ISSUE THREAD

**Issue by Roy-H23** [strongbox-password-safe/Strongbox#461]
Dear Team,

Yubikey works perfectly fine to unlock the Database on Mac and iOS.

I followed the Helpdesk page for Autofill on iOS. Registered a a virtual key for Autofill.
But it seems not to work as you can see in the Screenshots. Restarted the phone and the phrase is also correct. 
Any suggestion? Running on latest release.

Thank you!

![8ACCA57F-7A9A-4BEE-AF35-9900F5DF2B55](https://user-images.githubusercontent.com/69594894/106358069-32522880-630a-11eb-97cb-e0991e480161.png)
![1A717DCD-7702-43D4-8878-5034ECD8827C](https://user-images.githubusercontent.com/69594894/106358071-34b48280-630a-11eb-862e-32476b71fc72.png)
![ABE118F9-5A2F-43E7-B6E6-81D05C9E094D](https://user-images.githubusercontent.com/69594894/106358072-35e5af80-630a-11eb-996d-48bb9f2f4597.png)

**Comment by strongbox-mark** [comment]
Hi, You may need to initialize this manually in AutoFill once. If instead of tapping on a username suggestion in the QuickType bar above the keyboard, instead you should tap the "Key" icon and select Strongbox.

You should no be presented with your Databases List. If you tap on one of those you should be presented with the Manual Unlock screen. Can you see what Virtual Hardware Keys are listed there?

If you select the correct one, does it work?

Doing this once should be enough to get back to normal behaviour.

**Comment by Roy-H23** [comment]
Hi Mark,

that was a very nice tipp. BUT I had to disable FaceID for this manual activity. Afterwards I was able to select the Autofill Key and manually entered the password. Reenabling FaceID again and it works.

If FaceID is active and you try to select a database via the key symbol. It automatically prompts for FaceID or fails with the same error above. Maybe this is worth a mention in the Help Dokument for Yubikey Autofill.

Thanks for your help!
